lib/Elastica/Index.php

```diff
@@ -192,23 +192,6 @@ public function deleteDocuments(array $docs)
     }
 
     /**
-     * Optimizes search index.
-     *
-     * Detailed arguments can be found here in the link
-     *
-     * @param array $args OPTIONAL Additional arguments
-     *
-     * @return array Server response
-     *
-     * @deprecated Replaced by forcemerge
```

Comment:
We cannot remove this yet as it was never really released as deprecated. I suggest to keep this in for the next major release and only remove it afterwards.